cluster/addons/cluster-monitoring/google/heapster-controller.yaml

```diff
@@ -16,26 +16,26 @@
 apiVersion: extensions/v1beta1
 kind: Deployment
 metadata:
-  name: heapster-v1.1.0.beta2
+  name: heapster-v1.1.0
```

Comment:
Can you please bum it also in:
https://github.com/kubernetes/kubernetes/blob/master/test/kubemark/resources/heapster_template.json